lib/AST/Module.cpp

```diff
@@ -365,6 +365,12 @@ ModuleDecl::ModuleDecl(Identifier name, ASTContext &ctx)
   setAccess(AccessLevel::Public);
 }
 
+bool ModuleDecl::isClangModule() const {
+  if (findUnderlyingClangModule())
+    return true;
+  return false;
```

Comment:
(I mainly did this instead of `return findUnderlyingClangModule()` because I thought the latter might be confusing in terms of the actual return type)

Reply:
In such situations I usually write it as:
```
return findUnderlyingClangModule() != nullptr;
```